app/components/object-detail-widget/template.hbs

```diff
@@ -41,7 +41,12 @@
         <div>
             <h3>Identifiers</h3>
             <ul>
-                {{#each objectData._source.identifiers as |identifier|}}
+                {{#each identifiers.links as |identifier|}}
```

Comment:
Identifiers aren't ordered; I think contributors are the only field where order matters.